var/spack/repos/builtin/packages/py-cffi/package.py

```diff
@@ -33,15 +33,8 @@ class PyCffi(PythonPackage):
     # ./spack-src/cffi/ffiplatform.py has _hack_at_distutils which imports
     # setuptools before distutils, but only on Windows. This could be made
     # unconditional to support Python 3.12
```

Comment:
This comment can also be removed. Was this hack previously Windows-only but recently made platform-agnostic? Maybe the version ranges need to be changed.